### PR TITLE
Add OpInsel

### PR DESCRIPTION
**Pull request by Nils-afk**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/opinsel/manifest.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "server_wildcards": [
+      "%.opinsel.de"
+    ],
+    "supported_languages": [
+      "de"
+    ],
+    "social": {
+      "web": "https://opinsel.de.de",
+      "web_shop": "https://shop.opinsel.de",
+      "discord": "https://opinsel.de/discord"
```
Comment (maaaathis) [inline minecraft_servers/opinsel/manifest.json:14]: Please use a direct discord.gg/XYZ link here.
See: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url

File: minecraft_servers/opinsel/manifest.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+    "server_name": "opinsel",
+    "nice_name": "OpInsel",
+    "direct_ip": "minevale.de",
```
Comment (maaaathis) [inline minecraft_servers/opinsel/manifest.json:4]: Sure that `minevale.de` is your direct ip?

File: minecraft_servers/opinsel/manifest.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "nice_name": "OpInsel",
+    "direct_ip": "minevale.de",
+    "server_wildcards": [
+      "%.opinsel.de"
+    ],
+    "supported_languages": [
+      "de"
+    ],
+    "social": {
+      "web": "https://opinsel.de.de",
```
Comment (maaaathis) [inline minecraft_servers/opinsel/manifest.json:12]:
```suggestion
      "web": "https://opinsel.de",

```

File: minecraft_servers/opinsel/manifest.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      "web_shop": "https://shop.opinsel.de",
```
Comment (maaaathis) [inline minecraft_servers/opinsel/manifest.json:13]: This subdomain doesn't currently have a valid SSL certificate.

**Comment by toohard2explain** [comment]
image
does the server exist?

**Comment by Nils-afk** [comment]
> image does the server exist?

Yes, but my hoster is down at this moment.
I fix it today in the evening. I used a Server tamplate. I didn't know how to structure the file and that's why it said monevale.de

**Comment by Nils-afk** [comment]
> > image does the server exist?
> 
> Yes, but my hoster is down at this moment. I fix it today in the evening. I used a Server tamplate. I didn't know how to structure the file and that's why it said monevale.de

By the way the Server IP is mc.opinsel.de.

**Comment by DoJapHD** [comment]
PR inactive

**Pull request closed without merging**